Source/WebKit/WebProcess/Plugins/PDF/PDFPluginBase.h

```diff
@@ -100,6 +100,11 @@ class PDFPluginBase : public ThreadSafeRefCountedAndCanMakeThreadSafeWeakPtr<PDF
 
     virtual ~PDFPluginBase();
 
+    using ThreadSafeRefCountedAndCanMakeThreadSafeWeakPtr::ref;
+    using ThreadSafeRefCountedAndCanMakeThreadSafeWeakPtr::deref;
+    void refPDFScriptEvaluatorClient() const final { ref(); }
+    void derefPDFScriptEvaluatorClient() const final { deref(); }
+
```

Comment:
```
void ref() const final { ThreadSafeRefCountedAndCanMakeThreadSafeWeakPtr::ref(); }
void deref() const final { ThreadSafeRefCountedAndCanMakeThreadSafeWeakPtr::deref(); }
```

Reply:
Basically, "refXXX" is an anti-pattern. You can just call it "ref", with a pure virtual version in the base class and a concrete final version in the subclass.

Reply:
Oh I see--I used "refXXX" because I saw that pattern being used in a different class. But it is actually a lot clearer to use a pure virtual ref. Thank you for the suggestion, fixed!